train-1/HW3-268.cpp

```diff
@@ -0,0 +1,19 @@
+class Solution {
+public:
+    int missingNumber(vector<int>& nums) {
+        vector <int>:: iterator e;
```

Comment:
It's hard to remember this type name right? So you can: `decltype(nums.begin()) e;`